libc/socket/unix-socket.c

```diff
@@ -930,6 +931,14 @@ void runner(void)
 
 int main(int argc, char *argv[])
 {
+	/* Custom value (ms) added to timeout in assert since blocking interval
+	 * significantly overruns this timeout due to big kernel scheduling delays on some targets.
+	 * Typical for emulated targets.
+	 */
+	if (argc == 2) {
```

Comment:
IMHO would be better to add some option param to possibly not break `UnityMain` parsing `argv` parsing? (see how `UnityMain` parses options without the usage of `getopt`)

(eg. calling `test-libc-unix-socket --extra-poll-delay-ms 100` is more descriptive and needs much less comments)